app/web/components/ProfileIncompleteDialog/ProfileIncompleteDialog.tsx

```diff
@@ -0,0 +1,79 @@
+import { Typography } from "@material-ui/core";
+import Button from "components/Button";
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "components/Dialog";
+import StyledLink from "components/StyledLink";
+import { Trans, useTranslation } from "i18n";
+import { DASHBOARD } from "i18n/namespaces";
+import Link from "next/link";
+import React from "react";
+import { routeToEditProfile } from "routes";
+
+export interface ProfileIncompleteDialogProps {
+  open: boolean;
+  onClose: () => void;
+  attempted_action: "create_event" | "send_message" | "send_request";
+}
+
+export default function ProfileIncompleteDialog({
+  open,
+  onClose,
+  attempted_action,
+}: ProfileIncompleteDialogProps) {
+  const { t } = useTranslation([DASHBOARD]);
+
+  const action_text = t(
+    `dashboard:complete_profile_dialog.actions.${attempted_action}`
+  );
+
+  return (
+    <Dialog
+      aria-labelledby="profile-incomplete-dialog-title"
+      open={open}
+      onClose={onClose}
+    >
+      <DialogTitle id="profile-incomplete-dialog-title">
+        {t("dashboard:complete_profile_dialog.title")}
+      </DialogTitle>
+      <DialogContent>
+        <DialogContentText>
+          <Typography variant="body1" paragraph>
+            <Trans i18nKey="dashboard:complete_profile_dialog.description_1">
+              Before you can {{ action_name: action_text }}, you must{" "}
+              <strong>write a bit about yourself</strong> in your profile and{" "}
+              <strong>upload a profile photo</strong>.
+            </Trans>
+          </Typography>
+          <Typography variant="body1" paragraph>
+            <Trans i18nKey="dashboard:complete_profile_dialog.description_2">
+              This helps build a trusted community and reduce spam. For more
+              information,{" "}
+              <StyledLink
+                variant="body1"
+                href="https://help.couchers.org/hc/couchersorg-help-center/articles/1725919152-why-do-i-need-to-complete-my-profile-to-use-some-features"
+              >
+                please refer to this help page
+              </StyledLink>
+              . Thank you for your help!
+            </Trans>
+          </Typography>
+        </DialogContentText>
+      </DialogContent>
```

Comment:
This bit is causing a console error:

`
ProfileIncompleteDialog.tsx:23 Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    at p
    at Typography (webpack-internal:///./node_modules/@material-ui/core/esm/Typography/Typography.js:166:28)
    at WithStyles (webpack-internal:///./node_modules/@material-ui/styles/esm/withStyles/withStyles.js:61:31)
    at p
    at Typography (webpack-internal:///./node_modules/@material-ui/core/esm/Typography/Typography.js:166:28)
    at WithStyles (webpack-internal:///./node_modules/@material-ui/styles/esm/withStyles/withStyles.js:61:31)
    at DialogContentText
    at WithStyles (webpack-internal:///./node_modules/@material-ui/styles/esm/withStyles/withStyles.js:61:31)
    at DialogContentText (webpack-internal:///./components/Dialog.tsx:164:19)
    at div
    at DialogContent (webpack-internal:///./node_modules/@material-ui/core/esm/DialogContent/DialogContent.js:43:23)
    at WithStyles (webpack-internal:///./node_modules/@material-ui/styles/esm/withStyles/withStyles.js:61:31)
    at DialogContent (webpack-internal:///./components/Dialog.tsx:147:19)
    at div
    at Paper (webpack-internal:///./node_modules/@material-ui/core/esm/Paper/Paper.js:48:23)
    at WithStyles (webpack-internal:///./node_modules/@material-ui/styles/esm/withStyles/withStyles.js:61:31)
    at div
`

I think Typography must render a `<p>` under the hood and so must `DialogTextContent`.

I think you can just remove `<DialogTextContent>` here. That removes the error and doesn't change how it looks.